examples/model_selection/plot_tuned_decision_threshold.py

```diff
@@ -0,0 +1,378 @@
+"""
+==================================================
+Post-tuning the cut-off point of decision function
+==================================================
```

Comment:
I find "post-tuning" a bit misleading because in this example we fit the classifier and tune the threshold at the same time

Reply:
Post hoc tuning? Tuning after fit(ting)?